src/components/MenuBar/TimestampQueryBox.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+    Box,
+    useColorScheme,
+} from "@mui/joy";
+
+import SearchIcon from "@mui/icons-material/Search";
+
+import useUiStore from "../../stores/uiStore";
+import {THEME_NAME} from "../../typings/config";
+import {UI_ELEMENT} from "../../typings/states";
+import {isDisabled} from "../../utils/states";
+import {updateWindowUrlHashParams} from "../../utils/url";
+import {updateViewHashParams} from "../../utils/url/urlHash";
+import MenuBarIconButton from "./MenuBarIconButton";
+
+import "./TimestampQueryBox.css";
+
+
+/**
+ * Perform timestamp query with the datetime string.
+ *
+ * @param datetime
+ */
+const handleTimestampQuery = (datetime: string) => {
+    if (datetime) {
+        const timestamp = new Date(`${datetime}Z`).getTime();
+        updateWindowUrlHashParams({timestamp: timestamp});
+        updateViewHashParams();
+    }
+};
+
+
+/**
+ * renders a timestamp input field and a search icon button.
+ * Users can input a date and time in UTC format and either press "Enter"
+ * or click the search button to update the application's state and URL hash parameters.
+ *
+ * @return
+ */
+const TimestampQueryBox = () => {
+    const uiState = useUiStore((state) => state.uiState);
+    const {mode, systemMode} = useColorScheme();
+    return (
+        <Box className={"timestamp-query-box"}>
+            <input
```

Comment:
can we change the tooltip of the calendar button from "Show local date and time picker" -> "Show UTC picker" / "Show picker" so it's less misleading?